packages/tabs/tab.md

```diff
@@ -107,3 +92,31 @@ When an `<sp-tab>` element is given the `disabled` attribute it will prevent vis
     <sp-tab-panel value="4">Content for Tab 4 is selectable</sp-tab-panel>
 </sp-tabs>
 ```
+
+### Behaviors
+
+#### Selection
+
+- Clicking a tab selects it and shows its associated panel
+- Only one tab can be selected at a time within a tab group
+- Disabled tabs cannot be selected
+
+#### Keyboard Navigation
+
+- `Tab`: Move focus to the next focusable element
+- `Arrow Keys`: Navigate between tabs in the group
+- `Enter` or `Space`: Select the focused tab
+
```

Comment:
I'd move the keyboard navigation section to accessibility to be consistent with other components that include this information.